src/modules/task.js

```diff
@@ -0,0 +1,5 @@
+export default function Task(description, completed = false, index) {
```

Comment:
Here I found only seperate file for export is created which you can be done in your task collection .js file itself and the import into your index.js for best practices javascript.